bluepyemodel/efeatures_extraction/efeatures_extraction.py

```diff
@@ -28,6 +29,50 @@
 logger = logging.getLogger(__name__)
 
 
+def interpolate_RMP(fitness_calculator_configuration):
+    """If we do not have recordings with no holding, we need to estimate the RMP as:
+    RMP = V_hold - R_in*I_Hold
+    """
+    rin = None
+    holding_current = None
+    holding_voltages = []
+
+    for f in fitness_calculator_configuration.efeatures:
+        if (
+            f.efel_feature_name == "ohmic_input_resistance_vb_ssse"
+            and f.protocol_name == "RinProtocol"
+        ):
+            rin = f.mean
+        if f.efel_feature_name == "voltage_base" and (
+            "IV" in f.protocol_name or "IDrest" in f.protocol_name
```

Comment:
Do you want to take into consideration different variations of IDrest e.g IDRest, Idrest, idrest ? IDthresh and IDthres and their variations can also be included if user decides to use that. 
Maybe convert and search for lowercase protocols.

Reply:
done in next commit